SampleBase/src/Android/SampleAppAndroid.cpp

```diff
@@ -38,14 +38,16 @@ namespace Diligent
 class SampleAppAndroid final : public SampleApp
 {
 public:
+    std::atomic_bool paused;
```

Comment:
This should be a private member. Please name it `m_Paused` to be consistent with the conventions used in Diligent. Please also default-initialize the member `m_Paused{false};`

Reply:
Please add <atomic> header